<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ErrPage</class>
 <widget class="QDialog" name="ErrPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QPushButton" name="Main_ErrPage_btClose">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>100</y>
     <width>89</width>
     <height>25</height>
    </rect>
   </property>
   <property name="text">
    <string>PushButton</string>
   </property>
  </widget>
  <widget class="QLabel" name="Main_ErrPage_lbDep">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>170</y>
     <width>181</width>
     <height>61</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
  <widget class="QLabel" name="Main_ErrPage_lb1">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>260</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>TextLabel</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
